v7/wordpress_compiler/wordpress/wordpress.py

```diff
@@ -190,7 +190,14 @@ def __formatData(self, data, context, source=None):
             _LOGGER.warning("The post '" + source + "' still contains shortcodes: " + str(left_shortcodes))
         return output
 
+    def compile_string(self, data, source_path=None, is_two_file=True, post=None, lang=None):
+        """New interface. Might be removed at some time."""
```

Comment:
Oh come on, we are not that unstable. (this docstring could be more helpful anyways)